index.bs

```diff
@@ -8,46 +8,222 @@ Repository: w3c/webrtc-dscp-exp
 
-IDL specification {#idl}
-====
+In addition, this specification adds fields to {{RTCRtpEncodingParameters}}
+that allow control over the DSCP markings without affecting local
+prioritization, and vice versa.
+
+# Priority and QoS Model {#model}
+
+Many applications have multiple media flows of the same data type and
+often some of the flows are more important than others. WebRTC uses the
+priority and Quality of Service (QoS) framework described in
+[[!RTCWEB-TRANSPORT]] and [[!TSVWG-RTCWEB-QOS]] to provide priority and
+DSCP marking for packets that will help provide QoS in some networking
+environments. The priority setting can be used to indicate the relative
+priority of various flows. The priority API allows the JavaScript
+applications to tell the browser whether a particular media flow is high,
+medium, low or of very low importance to the application by setting the
+<code>priority</code> property of
+<code><a>RTCRtpEncodingParameters</a></code> objects to one of the
+values defined below.
+
+## RTCPriorityType Enum {#rtc-priority-type}
+<pre class="idl">
+enum RTCPriorityType {
+  "very-low",
+  "low",
+  "medium",
+  "high"
+};
+</pre>
+          <table data-link-for="RTCPriorityType" data-dfn-for="RTCPriorityType"
+          class="simple">
+            <tbody>
+              <tr>
+                <th colspan="2">Enumeration description</th>
+              </tr>
+              <tr>
+                <td><dfn data-idl><code>very-low</code></dfn></td>
+                <td>See [[!RTCWEB-TRANSPORT]], Sections 4.1 and 4.2. Corresponds to "below
+                normal" as defined in [[!RTCWEB-DATA]].</td>
+              </tr>
+              <tr>
+                <td><dfn data-idl><code>low</code></dfn></td>
+                <td>See [[!RTCWEB-TRANSPORT]], Sections 4.1 and 4.2. Corresponds to
+                "normal" as defined in [[!RTCWEB-DATA]].</td>
+              </tr>
+              <tr>
+                <td><dfn data-idl><code>medium</code></dfn></td>
+                <td>See [[!RTCWEB-TRANSPORT]], Sections 4.1 and 4.2. Corresponds to "high"
+                as defined in [[!RTCWEB-DATA]].</td>
+              </tr>
+              <tr>
+                <td><dfn data-idl><code>high</code></dfn></td>
+                <td>See [[!RTCWEB-TRANSPORT]], Sections 4.1 and 4.2. Corresponds to "extra
+                high" as defined in [[!RTCWEB-DATA]].</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </section>
+      <p>Applications that use this API should be aware that often better
+      overall user experience is obtained by lowering the priority of things
+      that are not as important rather than raising the priority of the things
+      that are.</p>
+    </section>
+
+## Extension to EncodingParameters {#encoding-parameters}
 
 <pre class='idl'>
 partial dictionary RTCRtpEncodingParameters {
-    RTCPriorityType networkPriority;  // Note: No default
+  RTCPriorityType priority = "low";
```

Comment:
Should remove default here; precedence needs to be explicit, and easier to specify with no default.